### PR TITLE
Fixed typos and Hive release version about Flink

### PR DESCRIPTION
**Pull request by 911432**
Fixed typos and Hive release version about Flink

### CODE REVIEW

File: docs/flink-getting-started.md [hunk 1]
```diff
@@ -221,7 +221,6 @@ The following properties can be set if using the Hive catalog:
 * `clients`: The Hive metastore client pool size, default value is 2. (Optional)
 * `warehouse`: The Hive warehouse location, users should specify this path if neither set the `hive-conf-dir` to specify a location containing a `hive-site.xml` configuration file nor add a correct `hive-site.xml` to classpath.
 * `hive-conf-dir`: Path to a directory containing a `hive-site.xml` configuration file which will be used to provide custom Hive configuration values. The value of `hive.metastore.warehouse.dir` from `<hive-conf-dir>/hive-site.xml` (or hive configure file from classpath) will be overwritten with the `warehouse` value if setting both `hive-conf-dir` and `warehouse` when creating iceberg catalog.
-* `hadoop-conf-dir`: Path to a directory containing `core-site.xml` and `hdfs-site.xml` configuration files which will be used to provide custom Hadoop configuration values.
```
Comment (hililiwei) [inline docs/flink-getting-started.md:224]: Why delete it? We do support this.

Comment (911432) [reply docs/flink-getting-started.md:224]: I misread hive-conf-dir as hadoop-conf-dir.

**Comment by 911432** [comment]
Iwi

**Comment by 911432** [comment]
I will update rebase master.

**Comment by 911432** [comment]
I will update rebase master. #7244

**Comment by 911432** [comment]
I will update rebase master. https://github.com/apache/iceberg/pull/7245

**Comment by 911432** [comment]
I have updated rebase at https://github.com/apache/iceberg/pull/7245

**Comment by 911432** [comment]
I have updated rebase at https://github.com/apache/iceberg/pull/7245

**Comment by 911432** [comment]
I have updated rebase at https://github.com/apache/iceberg/pull/7247

File: docs/flink-getting-started.md (hunk 1 above)
Comment (911432) [reply docs/flink-getting-started.md]: It's my mistake.

**Pull request closed without merging**